examples/flutter_view/lib/main.dart

```diff
@@ -47,13 +50,13 @@ class _MyHomePageState extends State<MyHomePage> {
   }
```

Comment:
You may consider making `handlePlatformIncrement` private. And perhaps merge it with `_incrementCounter` to keep the number of methods small.